aspnetcore/includes/razor.md

```diff
@@ -1,4 +1,10 @@
-This tutorial teaches ASP.NET Core MVC web development with controllers and views. Razor Pages is a new alternative in ASP.NET Core 2.0 and later, a page-based programming model that makes building web UI easier and more productive. We recommend you try the [Razor Pages](xref:tutorials/razor-pages/razor-pages-start) tutorial before the MVC version. The Razor Pages tutorial:
+This tutorial teaches ASP.NET Core MVC web development with controllers and views. Razor Pages is a feature of the ASP.NET Core MVC framework introduced in ASP.NET Core 2.0. Razor Pages programming:
```

Comment:
I'm not sure you need to say when it was introduced, and "page-based programming model" doesn't seem very helpful, and "is similar to MVC" is implied by the fact that it's a feature of MVC.  Here's a proposed rewording:

... Razor Pages is a feature of the ASP.NET Core MVC framework that makes building web UI easier and more productive.  You can use Razor pages alongside controllers and views in the same project.